packages/amplify_auth_plugin_interface/lib/src/SignUp/SignUpRequest.dart

```diff
@@ -25,10 +25,8 @@ class SignUpRequest {
     final Map<String, dynamic> pendingRequest = {
       'username': username,
       'password': password,
+      'options': options?.serializeAsMap()
```

Comment:
Nit: I noticed this pattern in one of @dnys1's PRs that I thought was pretty sweet
```suggestion
      if (options != null) 'options': options.serializeAsMap()
```
Then we can even remove line 30 while still maintaining a really concise yet expressive pattern.

Reply:
```suggestion
      if (options != null) 'options': options!.serializeAsMap()
```